doc/source/user_guide/groupby.rst

```diff
@@ -358,10 +361,12 @@ More on the ``sum`` function and aggregation later.
 
 Grouping DataFrame with Index levels and columns
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-A DataFrame may be grouped by a combination of columns and index levels by
-specifying the column names as strings and the index levels as ``pd.Grouper``
+A DataFrame may be grouped by a combination of columns and index levels. You
+need to specify the column names as strings, and the index levels as ``pd.Grouper``
 objects.
```

Comment:
can you change `''pd.Grouper''` to `:class:'Grouper'`.

Reply:
The index levels can also be specified by their labels, the columns can also be specified in a pd.Grouper, and columns need not be strings.

Reply:
"can you change ''pd.Grouper'' to :class:'Grouper'." -> Yes, I will.

I need help to rewrite the last part please. I'm not sure how to include the comment "The index levels can also be specified by their labels, the columns can also be specified in a pd.Grouper, and columns need not be strings."

Reply:
I think something like "You can specify both column and index names, or use a :class:`Grouper` object".

Reply:
Thanks